data/test/scenarios/wml_tests/FilterWML/filter_adjacent_location_count_two_active_bug.cfg

```diff
@@ -0,0 +1,101 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: ability[filter_adjacent_location]adjacent,count=
+##
+# Actions:
+# Give Alice an ability specialX, which is only active if exactly two of the adjacent units have specialY.
+# Give Charlie and Dave has specialY.
+# Test whether the ability is active.
+##
+# Expected end state:
+# Although specialX should be active, it isn't.
+#####
+{COMMON_KEEP_A_B_C_D_UNIT_TEST "filter_adjacent_location_count_two_active_bug" (
+    # Although this ability is meant to be active when the count of adjacent units with specialY is
+    # zero, the C++ implementation in abilities.cpp returns false (in several places) unless each
+    # direction listed in adjacent= contains a unit with that ability.
+    #
+    # It would be good to fix it before 1.20, but would be a behavior change so can't be backported
+    # to 1.18. When fixing the bug, this test should be renamed without the "_bug" suffix.
+    [event]
+        name=start
+
+        [object]
+            silent=yes
+            [effect]
+                apply_to=attack
+                [set_specials]
+                    mode=replace
+                    [damage]
+                        id=specialX
+                        name=_ "specialX"
+                        description=_ "specialX is active if and only if no adjacent units have specialY"
```

Comment:
This one should say "exactly two".

Found it while checking that the fix actually works.

Reply:
but it is encoded like if "count" is mandatory, and when i try without encode count=2(or other value) it fail a new, the 413 line it is really useful?